.github/workflows/LinuxRelease.yml

```diff
@@ -259,6 +259,10 @@ jobs:
         path: |
           libduckdb-linux-i386.zip
           duckdb_cli-linux-i386.zip
+    - uses: actions/upload-artifact@v2
```

Comment:
Not sure if we should just have a separate zip for all of the 32bit/i386 artefacts instead?

@hannes / @Mytherin ?